src/NodeServices/ExecutionSession.cs

```diff
@@ -69,5 +69,11 @@ public class ParameterKeys
         /// The duration of an execution covered by an <see cref="IExecutionSession"/>
```

Comment:
I think we should make this class static as well.

Reply:
I agree, but I think that would sadly break the API. Do you prefer I obsolete this class, create a static version and use that one?

Reply:
or just risk the change?

Reply:
Can we add a TODO to add it later?

Reply:
yep, will do.

Reply:
added and task created.